packages/react/src/components/Menu/Menu.tsx

```diff
@@ -203,6 +207,7 @@ class Menu extends AutoControlledComponent<ReactProps<MenuProps>, MenuState> {
           active,
           inSubmenu: submenu,
           indicator,
+          accessibility: accessibility ? (accessibility as any).childBehavior : undefined,
```

Comment:
```suggestion
          accessibility: accessibility ? accessibility.childBehavior : undefined,
```

Reply:
will add both the type change and this change. thanks!